packages/ShowMore/src/ShowMore.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import PropTypes from "prop-types";
+import truncate from "lodash/truncate";
+import uuid from "uuid/v4";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Button from "@paprika/button";
+import { toggleStyles } from "./ShowMore.styles";
+
+const propTypes = {
+  /** Additional description for "Show more" link. Should be a "topic" that will be appended to "Show more about [topic]". */
+  a11yText: PropTypes.string,
+
+  /* Full content to be revealed. */
+  children: PropTypes.node.isRequired,
+
+  /* Length, in characters, of truncated preview content. */
+  collapsedLength: PropTypes.number,
+};
+
+const defaultProps = {
+  a11yText: null,
+  collapsedLength: 255,
+};
+
+function getTruncatedVersion(content, length) {
+  return truncate(content, {
+    length,
+    separator: " ",
+  });
+}
+
+const ShowMore = props => {
```

Comment:
nip:  `function ShowMore(props) {

}`

better than assigning it to a variable.